web/frontend/src/pages/election/components/utils/useChangeAction.tsx

```diff
@@ -485,6 +488,19 @@ const useChangeAction = (
   };
 
   const getAction = () => {
+    if (Array.from(DKGStatuses.values()).includes(NodeStatus.Unreachable)) {
```

Comment:
The system must still work if there are more than 2/3 of the nodes cooperating (2f +1).

Reply:
So just to be 100% clear, strictly more than 2/3 of the nodes must be working ?

Reply:
It should, theoretically, but actually not in our implementation. The node that performs the setup waits on exactly the N node, otherwise it times out with an error: https://github.com/dedis/d-voting/blob/ee2c73d7f1b3d2e1eaa40bb4b2544dfcf2ebe03e/services/dkg/pedersen/mod.go#L235.

I'm adding that to the long list of things to improve 😅

Reply:
yes, sounds good. But if some nodes fails to initialize, do we have a way to initialize only the failed ones ?

Reply:
If I'm not mistaken the DKG service will return `{Status: 0, Error: {... Message:'actor already exists for electionID etc.'}}` if a node is already initialized ?

https://github.com/dedis/d-voting/blob/ee2c73d7f1b3d2e1eaa40bb4b2544dfcf2ebe03e/services/dkg/pedersen/mod.go#L103

So for the moment, all the nodes are re-initialized, but this error is ignored when we poll the DKG status (like when we try to setup twice the same node)

Reply:
Yes that would be great ! 
It's really not perfect, but I didn't have the time during the semester to re-implement all the promises, guess I'll leave that to our successors ...! 
I'll create another issue to record which parts of the code would have to be modified to be able to setup the nodes even if not all of them are initialized. Then sending only request to the nodes that are not initialized yet should be easy.